meowlflow/build.py

```diff
@@ -72,7 +72,7 @@
 EXPOSE 8000
 
 WORKDIR /opt/mlflow
-ENTRYPOINT ["python", "-c", "from mlflow.models import container as C; C._serve()"]
+ENTRYPOINT ["python", "-c", "from mlflow.models import container as C; C._serve("local")"]
```

Comment:
How come this is necessary now? It seems to work even without it, right?

Reply:
in all my recent installs it is necessary to add this `--env-manager ` argument.

Reply:
the recent model deployments at conny need it too (rental multiclass, address-ner...)